app/views/arclight/requests/_aeon_external_request_endpoint.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= form_tag @aeon_external_request.url, method: :post, class: 'al-request-form' do |f| %>
```

Comment:
This is going to need to be deprecated once the Request Cart Vue Component is integrated.